node/flags/flags.go

```diff
@@ -423,6 +430,31 @@ var (
 		Value:    16,
 		EnvVar:   common.PrefixEnvVar(EnvVarPrefix, "DOWNLOAD_POOL_SIZE"),
 	}
+	GetChunksHotCacheReadLimitMBFlag = cli.Float64Flag{
+		Name:     common.PrefixFlag(FlagPrefix, "get-chunks-hot-cache-read-limit-mb"),
+		Usage:    "The size of the hot cache read limit in megabytes / second.",
```

Comment:
The usage docs of these four new flags could benefit from some more detail. I wasn't sure how to interpret these until I went and looked at the docs in the `config` struct

Reply:
updated